icu4j/tools/build/build.xml

```diff
@@ -35,25 +24,11 @@
 
     <target name="clean" depends="@clean" description="Clean up the build outputs"/>
 
-    <target name="compile" depends="copy-src-pre8, @compile" description="Compile java source files"/>
+    <target name="compile" depends="@compile" description="Compile java source files"/>
 
     <target name="copy" depends="@copy" description="Copy non-java runtime files to the project's binary directory"/>
 
     <target name="jar" depends="compile, copy, @jar" description="Create the project's jar file"/>
 
-    <target name="src-jar" depends="copy-src-pre8, @src-jar" description="Create the project's source jar file"/>
```

Comment:
We used to support building docs with both JDK 7 and JDK 8. They need different version of source files. I completely dropped JDK 7 support, so no alternative source is required.

However, if we want to support newer version of JDK other than 8, we need to do something similar, because taglet API is not compatible.